README.md

```diff
@@ -16,32 +16,46 @@ This assumes that you have `Go` installed and setup.
 
 `pip3` and `python3` is what we need for development purpose.
 
-All dependecies are added in `dep` and we need to make sure we install `fresh` although it is already added to the lock.
 
+### How to install
 
-#### Development
-
-This is a service and it doesn't have any `UI` so in order to use this project or hack on the project we need a way to communicate with it. We can use the good old `curl` but let's be a little adventurous and use [POSTMAN](https://www.getpostman.com/) for the same.
-
-Make sure to carry on the following steps:
+1. [Install Go](https://golang.org/doc/install)
+2. Run the following commands
 
 ```bash
+$ make all
+$ dexer
+```
 
-make clean # for cleaning up
-make dev # for development purposes
-make logs # for getting the dummy directory ready
+Open http://localhost:8000 in your browser to open the user interface.
```

Comment:
Add a screenshot for the UI

Reply:
Use a service like https://imgbb.com/ for uploading the image and link it to the readme after it